adapters/Maio/MaioAdapter/RTBAdapter/GADRTBMaioInterstitialAd.m

```diff
@@ -0,0 +1,73 @@
+//
+//  GADRTBMaioInterstitialAd.m
+//  Adapter
+//
+//  Created by i-mobile on 2020/11/18.
+//  Copyright © 2020 Google. All rights reserved.
+//
+
+#import "GADRTBMaioInterstitialAd.h"
+#import "GADMMaioConstants.h"
+
+#import <MaioOB/MaioOB-Swift.h>
+
+#define MaioInterstitial MaioRewarded
+#define MaioInterstitialLoadCallback MaioRewardedLoadCallback
+#define MaioInterstitialShowCallback MaioRewardedShowCallback
+
+@interface GADRTBMaioInterstitialAd () <MaioInterstitialLoadCallback, MaioInterstitialShowCallback>
+
+@end
+
+@implementation GADRTBMaioInterstitialAd {
+  GADMediationInterstitialLoadCompletionHandler _completionHandler;
+  __weak id<GADMediationInterstitialAdEventDelegate> _adEventDelegate;
+  MaioInterstitial *_interstitial;
+}
+
+- (void)loadInterstitialForAdConfiguration:(nonnull GADMediationInterstitialAdConfiguration *)adConfiguration
+                       completionHandler:(nonnull GADMediationInterstitialLoadCompletionHandler)completionHandler {
+  _completionHandler = completionHandler;
```

Comment:
Can you wrap the `completionHandler` in a block similar to the way it is outlined in our CONTRIBUTING doc?
https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/CONTRIBUTING.md#best-practices